jobs/birthdayOfTheDay.ts

```diff
@@ -0,0 +1,40 @@
+import { TextChannel } from "discord.js"
+import moment from "moment";
+
+async function getBdayUsers(date) 
+{
+  const users = await prisma.user.findMany({
+    where: {
+        birthday: {
+          birthday: date
+        }
+    }
+  })
+  
+  const arrayIds = []
+  
+  for (let i = 0; i < users.length; i++)
+    arrayIds.push(users[i].discordId)
+
+  return arrayIds
+}
+
+const birthdayOfTheDay = {
+  cronPattern: '0 12 * * *', // every day at 12:00
+  channel: '💬｜general',
+  async execute(channel: TextChannel) {
+    const momentVariable = moment(new Date, 'MM/DD/YYYY')
+    var dateFormatted = momentVariable.format('MM/DD')
```

Comment:
make the format a const and put in the global scope of the file since it will never change